_sass/components/_list.scss

```diff
@@ -27,41 +27,32 @@
   background-size: 1rem;
 }
 
-.help-article > h2:first-of-type,
-.list-styled {
+.help-article > h2:first-of-type {
   margin-top: 2rem;
+}
 
-  & + ul, 
-  & + ol {
-    list-style-type: none;
-    margin-left: 0;
-    padding-left: 0;
+.help-article > h2:first-of-type + ul,
+.help-question-list,
+.page-content__prose ul.help-question-list {
+  list-style-type: none;
+  margin-left: 0;
+  padding-left: 0;
 
-    li:before {
-      content: "?";
-      background-color: $navy;
-      padding: 2px 8px;
+  li {
+    @include u-margin-y(1);
+
+    &::before {
+      background-color: color("primary-darker");
```

Comment:
I noticed you've replaced the custom color variables. I created _colors.scss to manage the USWDS design token color() in one place, and avoid directly using the token throughout styles. This way, if USWDS changes their token, we can change it in one location instead of looking all throughout the sass files. 
Can you explain your approach?

Reply:
> I noticed you've replaced the custom color variables. I created _colors.scss to manage the USWDS design token color() in one place, and avoid directly using the token throughout styles. This way, if USWDS changes their token, we can change it in one location instead of looking all throughout the sass files.
> Can you explain your approach?

Sure, my thinking is largely the inverse of the hypothetical you mention, which is: If Login wants to change its primary branding color from blue to magenta, a design token `primary` is flexible to this change, rather than needing to update all references to `$blue`.

Is there a history of USWDS changing design tokens?

Reply:
Revisiting color referencing isn't really something I want in scope here, so I restored these to the variables from `_colors.scss` in 36f96a1. We can continue the other discussion separately.